netlabel/labels_test.go

```diff
@@ -0,0 +1,69 @@
+package netlabel
+
+import (
+	"testing"
+
+	_ "github.com/docker/libnetwork/testutils"
+)
+
+func TestKeyValue(t *testing.T) {
+	input := []struct {
+		label string
+		key   string
+		value string
+		good  bool
+	}{
+		{"name=joe", "name", "joe", true},
+		{"age=24", "age", "24", true},
+		{"address:1234 First st.", "", "", false},
+		{"girlfriend=", "girlfriend", "", true},
```

Comment:
can you also add another case `{"key","key","", true}` ? 
A label without a value must work (even without a =).